getting_started/workflow/project_setup/version_control_systems.rst

```diff
@@ -35,3 +35,13 @@ There are some folders Godot creates you should have your VCS ignore:
   project, including sensitive information such as Android keystore credentials.
 - ``.mono/``: This folder stores automatically-generated Mono files. It only exists
   in projects that use the Mono version of Godot.
+
+Working with Git on Windows
+---------------------------
+
+Most Git for Windows clients are configured with the `core.autocrlf` set to `true`.
```

Comment:
Also, the syntax for inline code in reStructuredText uses two backticks instead of one.

```suggestion
Most Git for Windows clients are configured with the ``core.autocrlf`` set to ``true``.
```